src/explore-education-statistics-admin/src/pages/release/data/components/ReleaseDataUploadsSection.tsx

```diff
@@ -245,84 +246,115 @@ const ReleaseDataUploadsSection = ({
 
       <LoadingSpinner loading={isLoading}>
         {dataFiles.length > 0 ? (
-          <Accordion id="uploadedDataFiles">
-            {dataFiles.map(dataFile => (
-              <AccordionSection
-                id={dataFile.id}
-                key={dataFile.title}
-                heading={dataFile.title}
-                headingTag="h3"
-                open={dataFile.id === activeFileId}
-              >
-                <div style={{ position: 'relative' }}>
-                  {dataFile.isDeleting && (
-                    <LoadingSpinner text="Deleting files" overlay />
-                  )}
-                  <DataFileDetailsTable
-                    dataFile={dataFile}
-                    releaseId={releaseId}
-                    onStatusChange={handleStatusChange}
-                  >
-                    {canUpdateRelease &&
-                      terminalImportStatuses.includes(dataFile.status) && (
-                        <>
-                          {dataFile.status === 'COMPLETE' && (
-                            <>
-                              <Link
-                                className="govuk-!-margin-right-4"
-                                to={generatePath<ReleaseDataFileRouteParams>(
-                                  releaseDataFileRoute.path,
-                                  {
+          <EditingContextProvider editingMode="edit">
+            <EditableAccordion
```

Comment:
I'm not sure that reusing EditableAccordion is the best option here. Naming-wise it's confusing as it's not editable in this case and it ends up adding complexity to the EditableAccordion components to hide most of the functionality if it's not editable.

I suggest that we instead have ReorderableAccordion and ReorderableAccordionSection components that have the same reordering stuff as the Editable ones but with all the editing functionality taken out. This:
- would make the components simpler
- would be easier to reuse if we need more reorderable accordions in future, and
- wouldn't need to be inside the EditingContextProvider 

It'll be duplicating some code, which I'm guessing is what you were trying to avoid, but I think it's worth it here.

Happy to chat about this and help.